bsdploy/bootstrap_utils.py

```diff
@@ -281,16 +291,17 @@ def install_pkg(self, root, chroot=None, packages=[]):
         if pkg is not None:
             print("\nInstalling pkg")
             if not exists(pkg.local):
-                run('fetch -o {0.remote} {0.url}'.format(pkg))
+                run(self.env_vars + 'fetch -o {0.remote} {0.url}'.format(pkg), shell=False)
```

Comment:
How does this work? The shell for Fabric is `'/bin/sh`'. If we don't use it, is the login shell used? Can we be sure it's always ``csh`` needed for the ``setenv`` above? Wouldn't it be better to be explicit and just prepend ``http_proxy=... https_proxy=... {cmd}`` and use the shell?